trieste/objectives/single_objectives.py

```diff
@@ -587,3 +587,37 @@ def trid_10(x: TensorType) -> TensorType:
 The search space for :func:`trid` function is defined over :math:`[-d^2, d^2]` for all i=1,...,d.
 Here, we define it specifically for the 10-dimensional variant.
 """
+
+
+def beale(x: TensorType) -> TensorType:
```

Comment:
is there a specific reason why we need this one?

Reply:
Yes, for the advanced data transforms notebook. The Beale function is a bit more interesting than the Trid function since it has multiple modes as well as having high dynamic range. Updating the normalisation and model parameters improves BO performance for the Beale function, but doesn't for the Trid function.